docs/standard/serialization/snippets/system-text-json-converters-how-to/csharp/GetDefaultConverter.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace GetDefaultConverter;
+
+public class WeatherForecast
+{
+    public DateTime Date { get; set; }
+    public int TemperatureC { get; set; }
+    public string? Summary { get; set; }
+}
+
+// <Converter>
+public class MyCustomConverter : JsonConverter<int>
+{
+    private readonly static JsonConverter<int> s_defaultConverter = 
+        (JsonConverter<int>)JsonSerializerOptions.Default.GetConverter(typeof(int));
+
+    // Custom serialization logic
+    public override void Write(
+        Utf8JsonWriter writer, int value, JsonSerializerOptions options)
+    {
+        writer.WriteStringValue(value.ToString());
+    }
+
+    // Fall back to default deserialization logic
+    public override int Read(
+        ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        return s_defaultConverter.Read(ref reader, typeToConvert, options);
+    }
+}
+// </Converter>
+
+public class Program
+{
+    public static void Main()
```

Comment:
Nit: would it make sense to render new code samples using top-level statements?

Reply:
Yes, but this part of the code doesn't get rendered in the doc, only the actual converter code that is between the <Converter> tags is shown. This part of the code just makes it possible for the doc writer to test run the snippet. Including Program/Main lets me run a number of snippets in the same project, which I couldn't do with top-level statements. See, for example, https://github.com/dotnet/docs/blob/main/docs/standard/serialization/snippets/system-text-json-how-to-6-0/csharp/Program.cs